public_html/wp-content/themes/wordcamp-central-2012/style.css

```diff
@@ -1100,6 +1100,7 @@ li:after { content: "."; display: block; height: 0; clear: both; visibility: hid
 	margin-bottom: 9px;
 }
 #wc-content-blocks .wc-news h3 { background-position: 0 -71px }
+#wc-content-blocks .wc-global-sponsors h3 { background-position: 0 -147px; }
 #wc-content-blocks .wc-tweets h3 {
```

Comment:
There are still a few styles related to twitter (`wc-tweet`) in here, those could also be removed

Reply:
Good catch. Fixed with 09cade7f